### ISSUE TITLE
Make extensions field nullable

### ISSUE THREAD

**Issue by MrAolen** [Netflix/dgs-framework#35]
Hello,

I used the graphql client part of this library to consume a graphql API. When I received an error with this format : 

```json
{
  "data": {
    "result": null
  },
  "errors": [
    {
      "message": "some error message",
      "path": [
        "result"
      ],
      "locations": [
        {
          "column": 22,
          "line": 1
        }
      ]
    }
  ]
}
``` 
I got the following error message : 

`Instantiation of [simple type, class com.netflix.graphql.dgs.client.GraphQLError] value failed for JSON property extensions due to missing (therefore NULL) value for creator parameter extensions which is a non-nullable type`

We got this error when `extensions` field is not present, I assume it's not always the case. Is that possible to make it nullable ?

Best regards,

**Comment by tata406** [comment]
​{
  ​"​data​"​: {
    ​"​result​"​: ​null​
  },
  ​"​errors​"​: [
    {
      ​"​message​"​: ​"​some error message​"​,
      ​"​path​"​: [
        ​"​result​"​
      ],
      ​"​locations​"​: [
        {
          ​"​column​"​: ​22​,
          ​"​line​"​: ​1​
        }
      ]
    }
  ]
}

**Comment by paulbakker** [comment]
Fixed in #62
